lib/query/query_recurring_donations.rb

```diff
@@ -175,6 +189,11 @@ def self.get_chunk_of_export(npo_id, query, offset=nil, limit=nil, skip_header=f
     if include_stripe_customer_id
       select_list.push 'MAX(cards.stripe_customer_id) AS "Stripe Customer ID"'
     end
+
+    if include_last_failed_charge
+      select_list.push('MAX(failed_charges.created_at) AS "Last Failed Charge"')
+    end
```

Comment:
I think we should make sure that the last failed charge only includes the charges that happened during the month that CCR cares about. Wouldn't you agree?